<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Coding_FrameworkClass</class>
 <widget class="QMainWindow" name="Coding_FrameworkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Coding_Framework</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="MainTab">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="serial">
     <attribute name="title">
      <string>串口</string>
     </attribute>
     <widget class="QComboBox" name="SerialPortSelect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="BaudRateSelect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>5000000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <number>115200</number>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>300</y>
        <width>601</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>16</width>
        <height>451</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>601</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>40</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menu">
     <attribute name="title">
      <string>菜单</string>
     </attribute>
     <widget class="QPushButton" name="KEY_UP">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SUP</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_OK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SOK</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_DN">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SDN</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_RT">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SRT</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_LF">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SLF</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="menuOption0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>串口远程操作菜单</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="menuOption1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Option 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="menuOption2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Option 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="menuOption3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Option 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="menuOption4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Option 4</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="menuDisp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>240</width>
        <height>240</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>238</width>
         <height>238</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="KEY_LUP">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>LUP</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_LDN">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>LDN</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_LLF">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>LLF</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_LOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>LOK</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="KEY_LRT">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>LRT</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="image">
     <attribute name="title">
      <string>图像</string>
     </attribute>
     <widget class="QCheckBox" name="imageOption0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>使用当前串口数据</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="imageOption4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>保存原始图像</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="imageOption1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>使用文件夹内图片</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="imageOption2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>使用保存的串口数据</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="imageOption3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>保存图像至文件夹</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imgDirInput1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>400</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="imgDirSelect1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imagePrev">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>255</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imageNext">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>255</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageName">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>image name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imageSerialOnOff">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>225</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imageSerialRefresh">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>225</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="imageDisp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>320</width>
        <height>200</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>318</width>
         <height>198</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="imageIptDisp">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>160</width>
        <height>200</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>158</width>
         <height>198</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QToolButton" name="imgDirSelect2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imgDirInput2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>400</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="imgDirSelect3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>400</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imgDirInput3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>400</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="imageOption5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>430</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>保存透视图像</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageNumber">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>image number</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>BaudRateSelect</tabstop>
  <tabstop>imageOption4</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>SerialPortSelect</tabstop>
  <tabstop>imageOption0</tabstop>
  <tabstop>imgDirSelect1</tabstop>
  <tabstop>imageOption3</tabstop>
  <tabstop>imageIptDisp</tabstop>
  <tabstop>imageOption2</tabstop>
  <tabstop>imageOption1</tabstop>
  <tabstop>imagePrev</tabstop>
  <tabstop>imageNext</tabstop>
  <tabstop>imageSerialOnOff</tabstop>
  <tabstop>imageSerialRefresh</tabstop>
  <tabstop>imgDirInput1</tabstop>
  <tabstop>imageDisp</tabstop>
  <tabstop>imgDirSelect2</tabstop>
  <tabstop>imgDirInput2</tabstop>
  <tabstop>imgDirSelect3</tabstop>
  <tabstop>MainTab</tabstop>
  <tabstop>menuDisp</tabstop>
  <tabstop>menuOption0</tabstop>
  <tabstop>menuOption1</tabstop>
  <tabstop>menuOption2</tabstop>
  <tabstop>menuOption3</tabstop>
  <tabstop>menuOption4</tabstop>
  <tabstop>KEY_UP</tabstop>
  <tabstop>KEY_LF</tabstop>
  <tabstop>KEY_OK</tabstop>
  <tabstop>KEY_DN</tabstop>
  <tabstop>KEY_RT</tabstop>
  <tabstop>imgDirInput3</tabstop>
  <tabstop>imageOption5</tabstop>
 </tabstops>
 <resources>
  <include location="coding_framework.qrc"/>
 </resources>
 <connections/>
</ui>
